debian/changelog

```diff
@@ -1,6 +1,6 @@
-mercury (2.1.0~rc3-1) unstable; urgency=medium
+mercury (2.1.0~rc4-1) unstable; urgency=medium
   [ Alexander Oganezov ]
-  * Update to release v2.1.0rc3
+  * Update to release v2.1.0rc4
 
  -- Alexander Oganezov <alexander.a.oganezov@intel.com>  Wed, 10 Nov 2021 08:47:42 -0400
```

Comment:
Could make the date here more accurate if you need to push again for any reason.

Reply:
done